### PR TITLE
Gdr 2108

### PR DESCRIPTION
**Pull request by bczech**
# Description
## What changed?
Improved logic of standardize_MAE
Related JIRA issue: GDR-2108

## Why was it changed?
To be able to standardize and restore default identifiers based on metadata

# Checklist for sustainable code base
- [ ] I added tests for any code changed/added
- [ ] I added documentation for any code changed/added
- [ ] I made sure naming of any new functions is self-explanatory and consistent

# Logistic checklist
- [X] Package version bumped
- [X] Changelog updated

# Screenshots (optional)

### CODE REVIEW

**Review by gladkia — APPROVED** [review]

File: R/standardize_MAE.R [hunk 1]
```diff
@@ -32,14 +32,24 @@ standardize_mae <- function(mae) {
...
+  } else {
+    from_idfs <- idfs
+    to_idfs <- idfs_se
+  }
+  
   # Extract matching names of identifiers
-  matching_idfs <- .extract_matching_identifiers(idfs,
-                                                 idfs_se)
+  matching_idfs <- .extract_matching_identifiers(to_idfs,
+                                                 from_idfs)
```
Comment (j-smola) [inline R/standardize_MAE.R:52]: I like these names - easy and clear 😉

File: R/standardize_MAE.R [hunk 2]
```diff
@@ -32,14 +32,24 @@ standardize_mae <- function(mae) {
 #' standardize_se(se)
 #' 
 #' @export
-standardize_se <- function(se) {
+standardize_se <- function(se, use_default = TRUE) {
   checkmate::assert_class(se, "SummarizedExperiment")
+  
+  gDRutils::reset_env_identifiers()
```
Comment (j-smola) [inline R/standardize_MAE.R:38]: Why not?
```suggestion
 reset_env_identifiers()
```

**Review by j-smola — COMMENTED** [review]
Nicely done 👍 
Please, add tests & example for `use_default = FALSE` in `standardize_mae`

**Review by j-smola — APPROVED** [review]
Please, run `devtools::document()` before merging!

File: tests/testthat/test-standardize_MAE.R [hunk 3]
```diff
@@ -22,16 +22,23 @@ test_that("standardize_se works as expected",  {
...
-    "overwriting existing metadata entry: 'identifiers'"
-  )
-  expect_equal(lapply(get_SE_identifiers(se_standardized), sort),
-               lapply(get_SE_identifiers(se_original), sort))
+  se_standardized <- standardize_se(se)
   expect_equal(convert_se_assay_to_dt(se_standardized, "RawTreated"),
                convert_se_assay_to_dt(se_original, "RawTreated"))
 })
 
+test_that("standardize_se works as expected with default = FALSE",  {
```
Comment (j-smola) [inline tests/testthat/test-standardize_MAE.R:30]: 💪

**Review by j-smola — APPROVED** [review]
GJ!

**Pull request merged**